agent-inject/agent/container_volume.go

```diff
@@ -8,7 +8,6 @@ const (
 	configVolumeName    = "vault-config"
 	configVolumePath    = "/vault/configs"
 	secretVolumeName    = "vault-secrets"
-	secretVolumePath    = "/vault/secrets"
```

Comment:
This constant should remain and used if `AnnotationVaultSecretVolumePath` is not set.